okhttp-testing-support/src/main/kotlin/okhttp3/internal/io/InMemoryFileSystem.kt

```diff
@@ -119,4 +119,7 @@ class InMemoryFileSystem : FileSystem, TestRule {
   }
 
   override fun toString() = "InMemoryFileSystem"
+  fun allPaths(): MutableSet<File> {
+    return files.keys
```

Comment:
files.keys is a live view; `files.keys.toSet()` is a necessary defensive copy, if that’s what we want